src/Tabs/TabList.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext, useEffect, HTMLAttributes } from 'react';
+import styled from 'styled-components';
+import { TabContext, ContextProps } from './Tabs';
+
+const TabListComponent = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+interface OwnProps {
+  defaultTabId: string
+}
+
+type Props = OwnProps & HTMLAttributes<HTMLDivElement>
+
+const TabList: React.FC<Props> = ({ children, defaultTabId, ...props }) => {
```

Comment:
Is this intentional? It seems the only component that we can override styles on is `TabList` shouldn't `TabContent` at least also have this or not be a div?

Reply:
Yes, ...props is not required here. Removed now.